Liferay-iOS-SDK.podspec

```diff
@@ -25,4 +25,6 @@ Pod::Spec.new do |s|
 							}
 	s.requires_arc			= true
 	s.dependency			"AFNetworking", "= 2.6.3"
+	s.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => 'LIFERAY_MOBILE_SDK_MODULE=1' }
```

Comment:
can you rename `LIFERAY_MOBILE_SDK_MODULE` by `LIFERAY_MOBILE_SDK_FRAMEWORK`?

Reply:
Done! :D